2-mandatory/2-water-bottle.js

```diff
@@ -18,19 +18,19 @@ You have to implement the missing features according to the specification.
 let bottle = {
   volume: 0,
   fillUp: function () {
-    // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100;
   },
   pour: function () {
-    // calling this function should increase your bottle volume by 10 units;
+    (this.volume + 10) <= 100 ? this.volume += 10 : this.volume = 100;
   },
   drink: function () {
-    // calling this function should decrease your bottle volume by 10 units;
+    (this.volume - 10) >= 0 ? this.volume -= 10 : this.volume = 0;
   },
   isFull: function () {
-    // this function should return true if your bottle is full;
+    return this.volume === 100;
```

Comment:
nice - `x === y` evaluates to `true/false` anyway so this is a nice shorthand instead of `if (this.volume === 100) return true`